rascaline/src/calculators/soap/spherical_expansion.rs

```diff
@@ -314,7 +322,12 @@ impl SphericalExpansion {
     }
 
     /// Compute the contribution of a single pair and store the corresponding
-    /// data inside the given descriptor
+    /// data inside the given descriptor.
+    ///
+    /// This will store data both for the spherical expansion with `pair.first`
+    /// as the center and `pair.second` as the neighbor, and for the spherical
+    /// expansion with `pair.second` as the center and `pair.first` as the
+    /// neighbor.
```

Comment:
this is doing the calculation for a pair, not a single neighbour. I tried to clarify a bit